ar_tag: ar_marker_2
compute_best_spots: true
compute_relation_guards: true
debug: true
generate_guards: true
guards_limit: 50
guards_pcd: /data/final_guards
machine2ar_calibration: scie2ar6_calibration_mat
machine_mesh: /mesh/table_coro.obj
machine_pcd: /mesh/table_coro_fill.ply
observable_concavity: /data/observed_concavity
observed_pcd: /data/observed_pdc
potential_guard: 2
reachability_map: /mesh/reachability_map.pcd
relation_guards: /data/relation_path_arm_
relation_spot: /data/relation_coro.pkl
result_pcd: /data/final_result
scie2ar6_calibration_mat:
- - -0.9995857457968185
  - -0.027734899091643755
  - 0.00768844406850743
  - -0.8607933585359786
- - 0.027168053068934703
  - -0.9974501773171279
  - -0.06599273189127518
  - 0.030253419205516163
- - 0.009499141659211974
  - -0.06575651406823903
  - 0.9977904825990934
  - -1.0165813680281603
- - 0.0
  - 0.0
  - 0.0
  - 1.0
sort_spots: true
spot_limit: 25
spots: /data/relation_result_spots.pkls
spots_pcds: /data/pcd_17_04_saw
table2ar2_calibration_mat:
- - 0.9972773283665013
  - -0.06408157594352347
  - 0.03648947726082053
  - 0.08336138881899191
- - 0.06729985875326293
  - 0.9931861255790659
  - -0.09514225648488034
  - 0.9152489689435612
- - -0.030143976810705724
  - 0.09733895202763551
  - 0.994794686897853
  - -1.0366598910813083
- - 0.0
  - 0.0
  - 0.0
  - 1.0
trajectorie: /data/offline_trajectory_saw.pkl
